examples/apps/cli/Makefile.am

```diff
@@ -119,6 +119,36 @@ ot_cli_mtd_SOURCES                                                     = \
     $(SOURCES_COMMON)                                                    \
     $(NULL)
 
+if OPENTHREAD_ENABLE_RADIO_ONLY
+bin_PROGRAMS                                                          += \
+    ot-cli-radio                                                         \
+    $(NULL)
```

Comment:
I am not sure, but it kind of looks as if the `\` at the end here is not aligned with the rest of them below it?